cmd/ndm_daemonset/probe/addhandler.go

```diff
@@ -124,6 +124,7 @@ func (pe *ProbeEvent) addBlockDevice(bd blockdevice.BlockDevice, bdAPIList *apis
 			return nil
 		}
 	} else {
+		// FIXME: update controller cache after setting uuid
 		bd.UUID = uuid
```

Comment:
A change is needed in this file [here](https://github.com/openebs/node-disk-manager/blob/master/cmd/ndm_daemonset/probe/addhandler.go#L46-L47) about the log messages used. As change can also now trigger the adding device to cache

Reply:
done